packages/gas-fee-controller/package.json

```diff
@@ -33,8 +33,8 @@
     "@metamask/base-controller": "^3.2.3",
     "@metamask/controller-utils": "^5.0.2",
     "@metamask/eth-query": "^4.0.0",
-    "@metamask/network-controller": "^15.2.0",
-    "@metamask/polling-controller": "^1.0.1",
+    "@metamask/network-controller": "^16.0.0",
+    "@metamask/polling-controller": "^1.0.2",
```

Comment:
Should we add this to the changelog too?

Reply:
Fixed: https://github.com/MetaMask/core/pull/2044/commits/194c74e747285515e41738387b9b59d156b9b0fe